admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -387,7 +387,7 @@ module.exports = {
         test: /.(jpg|png|woff2?|eot|otf|ttf|svg|gif)$/,
         type: 'asset/resource',
         generator: {
-          filename: '[hash].[ext]',
+          filename: '[hash][ext]',
```

Comment:
Why do you need to change this? Doesn't it mean the generated files will look like `sqdfsdfwoff` instead of `sqdfsdf.woff` ?

Reply:
Ah ok, so I guess the `[ext]` includes the dot already